pkg/ddb/builder.go

```diff
@@ -32,7 +32,7 @@ func (b *keyBuilder) buildKey(item interface{}) (KeyValues, error) {
 
 func (b *keyBuilder) fromItem(item interface{}) (KeyValues, error) {
 	if item == nil {
-		return nil, fmt.Errorf("can not build key attributes from nil item")
+		return nil, fmt.Errorf("can not build key attributes from nil Item")
```

Comment:
is there a reason why "Item" is now ucfirst everywhere? :thinking:

Reply:
overeager refactoring of my IDE. Should be reverted